src/main/integrations/sentry-minidump/minidump-loader.ts

```diff
@@ -69,24 +65,23 @@ export function createMinidumpLoader(
             const twoSecondsAgo = new Date().getTime() - NOT_MODIFIED_MS;
 
             if (stats.mtimeMs < twoSecondsAgo) {
-              const file = await fs.readFile(path);
-              const data = preProcessFile(file);
-
-              if (data.length < 10_000 || data.subarray(0, 4).toString() !== MINIDUMP_HEADER) {
-                logger.warn('Dropping minidump as it appears invalid.');
+              const data = await fs.readFile(path);
+              try {
+                const parsedMinidump = parseMinidump(data);
+
+                logger.log('Sending minidump');
+
+                await callback(parsedMinidump, {
+                  attachmentType: 'event.minidump',
+                  filename: basename(path),
+                  data,
+                });
+              } catch (e) {
+                const message = e instanceof Error ? e.message : 'Unknown error';
+                logger.warn(`Dropping minidump: ${message}`);
```

Comment:
I wonder if we should serialize the entire error to get a stacktrace.

Reply:
Do all supported node versions have error.cause?

Reply:
Yup, we're 18+ now in `@sentry/node`, and [`error.cause` is `16.9.0`](https://nodejs.org/en/blog/release/v16.9.0#error-cause)